src/components/EditForm.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { deleteTodo, updateTodo } from '../api/todos';
+import { TodosContext } from '../TodosContext';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.